File: Modul_2/Bahasa Python/tempCodeRunnerFile.py
print("Masukkan tinggi dan alas segitiga")
import math
data = input().split()

if len(data) == 1:
    data.append(input())

A, B = map(float, data)

C = math.sqrt(B**2 - A**2)
keliling = A + B + C
luas = 0.5 * A * C

print(f"Alas = {C:.0f} cm")
print(f"Tinggi = {A:.0f} cm")
print(f"Keliling = {keliling:.0f} cm")
print(f"Luas = {luas:.0f} cm^2")

File: Modul_2/Bahasa C/tempCodeRunnerFile.c
#include <stdio.h>

int main() {
    double r, t;
    double volume, luas, keliling;
    double mpi = 22.0 / 7.0;
    int count;

    printf("Masukkan jari-jari dan tinggi tabung:\n");
    count = scanf("%lf %lf", &r, &t);

    if (count == 1) {
        scanf("%lf", &t);
    }

    volume = mpi * r * r * t;
    luas = 2 * mpi * r * (r + t);
    keliling = 2 * mpi * r;

    printf("Volume     = %.2lf\n", volume);
    printf("Luas       = %.2lf\n", luas);
    printf("Keliling   = %.2lf\n", keliling);

    return 0;
}
